packages/bento-design-system/src/Button/Button.tsx

```diff
@@ -8,6 +8,17 @@ import { Label } from "../Typography/Label/Label";
 import { Column, Columns } from "../Layout/Columns";
 import { IconProps } from "..";
 import { useBentoConfig } from "../BentoConfigContext";
+import pick from "lodash.pick";
+
+export type OtherButtonKeys =
```

Comment:
Can we extract the keys to a const and derive the type from it?

Something like

```ts
const otherButtonKeys = [...] as const

type OtherButtonKeys = typeof otherButtonKeys[number]
```

Reply:
Pushed, thats a clean pattern i didn't know you could extract those values in TS like that, very useful.